app/util/searchUtils.js

```diff
@@ -870,128 +814,45 @@ export const executeSearch = (getStore, refPoint, data, callback) => {
-      position.status !== 'geolocation-not-supported'
-    ) {
-      searchComponents.push(getCurrentPositionIfEmpty(input, position));
-    }
-    if (endpointLayers.includes('OldSearch')) {
-      const dropLayers = ['currentPosition'];
-      // old searches should also obey the layers definition
-      if (!endpointLayers.includes('FavouritePlace')) {
-        dropLayers.push('favouritePlace');
-      }
-      // searchComponents.push(getOldSearches(oldSearches, input, dropLayers));
-    }
-
-    if (endpointLayers.includes('Geocoding')) {
-      const focusPoint =
-        config.autoSuggest.locationAware && position.hasLocation
-          ? {
-              // Round coordinates to approx 1 km, in order to improve caching
-              'focus.point.lat': position.lat.toFixed(2),
-              'focus.point.lon': position.lon.toFixed(2),
-            }
-          : {};
-
-      const sources = get(config, 'searchSources', '').join(',');
-
-      searchComponents.push(
-        getGeocodingResult(
-          input,
-          config.searchParams,
-          language,
-          focusPoint,
-          sources,
-          config,
-        ),
-      );
-    }
-
-    if (endpointLayers.includes('Stops')) {
-      const focusPoint =
-        config.autoSuggest.locationAware && position.hasLocation
-          ? {
-              // Round coordinates to approx 1 km, in order to improve caching
-              'focus.point.lat': position.lat.toFixed(2),
-              'focus.point.lon': position.lon.toFixed(2),
-            }
-          : {};
-      const sources = get(config, 'feedIds', [])
-        .map(v => `gtfs${v}`)
-        .join(',');
-
-      if (sources) {
-        searchComponents.push(
-          getGeocodingResult(
-            input,
-            undefined,
-            language,
-            focusPoint,
-            sources,
-            config,
-          ),
-        );
-      }
-    }
-
-    endpointSearchesPromise = Promise.all(searchComponents)
-      .then(resultsArray => {
-        if (
-          endpointLayers.includes('Stops') &&
-          endpointLayers.includes('Geocoding')
-        ) {
-          // sort & combine pelias results into single array
-          const modifiedResultsArray = [];
-          for (let i = 0; i < resultsArray.length - 2; i++) {
-            modifiedResultsArray.push(resultsArray[i]);
-          }
-          const sorted = orderBy(
-            resultsArray[resultsArray.length - 1].concat(
-              resultsArray[resultsArray.length - 2],
-            ),
-            [u => u.properties.confidence],
-            ['desc'],
-          );
-          modifiedResultsArray.push(sorted);
-          return modifiedResultsArray;
+  const language = 'fi';
+  const searchComponents = [];
+  const focusPoint =
+    config.autoSuggest.locationAware && position.hasLocation
+      ? {
+          'focus.point.lat': position.lat.toFixed(2),
+          'focus.point.lon': position.lon.toFixed(2),
```

Comment:
This if clause will never evaluate to `true` because it looks at `position.hasLocation` and position [is declared](https://github.com/kyyticom/digitransit-ui/pull/6/files#diff-6200a9cfe751d20be96e10898bb53f76R818) above it as an empty object without such flag. As far as I understand this is used in widget so doesn't really matter and could be removed completely? Or what is the intended functionality?